apps/src/Sound.js

```diff
@@ -308,6 +316,7 @@ Sound.prototype.onSoundLoaded = function () {
   if (this.onLoad) {
     this.onLoad();
   }
+  this.isLoaded_ = true;
```

Comment:
I would move this line above the `onLoad()` callback. It seems better that if the callback code called the new `isLoaded()`, it would return `true` at this point.